### DISCUSSION TITLE
Typo not corrected?

### DISCUSSION THREAD

**Discussion by marcreverdin** [DivinumOfficium/divinum-officium#3525] [Q&A]
Hello, I’m new to this project, I use Divinum Officium a lot for prayer but noticed many typos in the French translation - which brought me here. I’m happy to help clean the whole French translation, however I did a few edits yesterday and today but they do not appear to be taken into account. I followed the instruction (“For small changes -- for example, for typographical corrections -- the simplest way to do so is to navigate to the relevant file in GitHub's repository browser and use its built-in editor. Any changes made in this way will automatically be converted to a pull request.”), but I may be wrong somewhere because my changes just disappear without any confirmation. I’d be grateful for any guidance so that my existing contributions are not lost, and so that I can help effectively in the future :) Best, Marc

**Comment by APMarcello3** [comment]
Nous avons reçu votre changement (pull request #3526). Assurez-vous de soumettre les changements comme pull request. Merci de nouveau pour votre collaboration.
